packages/amplication-server/src/core/serviceTopics/serviceTopics.service.ts

```diff
@@ -43,7 +44,7 @@ export class ServiceTopicsService extends BlockTypeService<
         project: {
           id: resource.projectId
         },
-        id: messageBrokerId,
+        id: { equals: messageBrokerId },
```

Comment:
see my comments above and revert this code please

Reply:
Done